Index: mp3splt-gtk/configure
===================================================================
--- mp3splt-gtk.orig/configure	2016-08-15 11:11:34.365706041 +0200
+++ mp3splt-gtk/configure	2016-08-15 11:11:49.537928546 +0200
@@ -15101,7 +15101,7 @@
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 
-  CFLAGS="$CFLAGS -DGTK_DISABLE_SINGLE_INCLUDES -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
+  CFLAGS="$CFLAGS -DGTK_DISABLE_SINGLE_INCLUDES -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
   #CFLAGS="$CFLAGS -DGTK_DISABLE_SINGLE_INCLUDES -DGSEAL_ENABLE"
 
 fi
Index: mp3splt-gtk/configure.ac
===================================================================
--- mp3splt-gtk.orig/configure.ac	2016-08-15 11:11:34.365706041 +0200
+++ mp3splt-gtk/configure.ac	2016-08-15 11:11:59.166069738 +0200
@@ -140,7 +140,7 @@
 
 PKG_CHECK_MODULES(GTK, [gtk+-3.0 >= 3.4.2],
 [
-  CFLAGS="$CFLAGS -DGTK_DISABLE_SINGLE_INCLUDES -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
+  CFLAGS="$CFLAGS -DGTK_DISABLE_SINGLE_INCLUDES -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
   #CFLAGS="$CFLAGS -DGTK_DISABLE_SINGLE_INCLUDES -DGSEAL_ENABLE"
 ],
 [
