Index: mp3splt-gtk/src/Makefile.am
===================================================================
--- mp3splt-gtk.orig/src/Makefile.am	2016-08-15 11:09:12.295621794 +0200
+++ mp3splt-gtk/src/Makefile.am	2016-08-15 11:09:12.275621501 +0200
@@ -52,7 +52,7 @@
 
 INCLUDES = @GTK_CFLAGS@ @LIBMP3SPLT_CFLAGS@
 
-mp3splt_gtk_LDADD = @GTK_LIBS@ -lgthread-2.0 @LIBMP3SPLT_LIBS@ -lm
+mp3splt_gtk_LDADD = @GTK_LIBS@ -lgthread-2.0 @LIBMP3SPLT_LIBS@ -lm -ldbus-glib-1
 
 # Define a C macro LOCALEDIR indicating where catalogs will be installed.
 localedir = $(datadir)/locale
